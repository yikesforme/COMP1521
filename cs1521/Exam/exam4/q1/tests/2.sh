sh exe tests/s2.s
