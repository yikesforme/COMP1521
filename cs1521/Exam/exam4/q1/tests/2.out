s1 = aeiouAEIOU
s1 = aeiouAEIOU
s2 = 
nv = 10
